doc/development.mdwn

```diff
@@ -0,0 +1,172 @@
+Developing LinkChecker
+======================
+
+The following steps describe how to compile LinkChecker from source
+on various platforms.
+
+This is a technical document, if you are looking for ways to
+participate in the community, you should rather look into
+[contributing](contributing).
+
+Requirements
+------------
+On Mac OS X systems, using MacPorts, Fink or homebrew for software
+installation is recommended.
+
+- Install Python >= 2.7.2 from http://www.python.org/
+
+- *On Windows only*, install the Windows SDK
+  http://msdn.microsoft.com/de-de/windows/bb980924
+
+- *On Windows only*, download and install the Microsoft
+  Visual C++ 2008 runtime from
+  http://www.microsoft.com/downloads/details.aspx?FamilyID=9b2da534-3e03-4391-8a4d-074b9f2bc1bf&displaylang=en
+
+- *Optional, used for Virus checking:*
+  ClamAv for Unix from http://www.clamav.net/lang/en/download/
+  or for Windows from http://www.sosdg.org/clamav-win32/
+
+- *Optional, for displaying country codes:*
+  Pygeoip from http://code.google.com/p/pygeoip/
```

Comment:
The url says "This project has been moved over to github and will be deleted soon: https://github.com/appliedsec/pygeoip"

But anyway https://pypi.python.org/pypi/pygeoip exists, so maybe we should just add it to our install_requires and/or requirements.txt and drop the mention here?